modules/KalturaSupport/components/chapters/chapters.js

```diff
@@ -971,18 +981,35 @@
 		},
 		//UI Handlers
 		mediaClicked: function (mediaIndex) {
-			if (this.getConfig("closeOnClick") && (this.getConfig('parent') === 'sideBarContainer')){
-				this.getPlayer().triggerHelper("closeSideBarContainer");
-			}
-			//Only apply seek in VOD or in live if DVR is supported
-			if ((this.getPlayer().isLive() && this.getPlayer().isDVR()) ||
-					!this.getPlayer().isLive()) {
+		//disable click from items out of DVR window
+		if (this.getComponent().find("li[data-mediaBox-index='" + mediaIndex + "']").hasClass("out-of-dvr")) {
+			return;
+		}
+		if (this.getConfig("closeOnClick") && (this.getConfig('parent') === 'sideBarContainer')){
+			this.getPlayer().triggerHelper("closeSideBarContainer");
+		}
+		//Only apply seek in VOD or in live if DVR is supported
+		if ((this.getPlayer().isLive() && this.getPlayer().isDVR()) ||
+			!this.getPlayer().isLive()) {
 				//Send play request on first click for devices that don't have autoplay, e.g. mobile devices
 				if (!this.getPlayer().canAutoPlay() && this.getPlayer().firstPlay){
 					this.getPlayer().sendNotification('doPlay');
 				}
-				// see to start time and play ( +.1 to avoid highlight of prev chapter )
-				this.getPlayer().sendNotification('doSeek', ( this.mediaList[mediaIndex].startTime ) + 0.1);
+				if (this.embedPlayer.isDVR()) {
+					// get current time from Id3
```

Comment:
I think this code should be in a generic place so can be reused on other services
Like mw.id3tag.js .absolute2relative(...)